web/profiles/custom/os2loop/themes/os2loop_theme/assets/components/pdf/pdf.scss

```diff
@@ -60,40 +116,44 @@ img {
 
 	.title-icon {
 		margin-right: 1em;
+        height: 30px;
+        width: 20px;
+      svg {
+        background: blue;
+        height: 20px;
+        width: 20px;
+      }
```

Comment:
Is this `svg` styling a leftover from a debugging session? Remove it if that's the case.